backend/src/main/java/de/grimsi/gameyfin/igdb/IgdbWrapper.java

```diff
@@ -17,35 +20,34 @@
 
 import javax.annotation.PostConstruct;
 import java.net.URI;
+import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static de.grimsi.gameyfin.igdb.IgdbApiProperties.GAME_QUERY_FIELDS_STRING;
+import static de.grimsi.gameyfin.igdb.IgdbApiQueryBuilder.*;
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
 @Slf4j
 @RequiredArgsConstructor
 @Service
 public class IgdbWrapper {
+    private final WebClient.Builder webclientBuilder;
+    private final WebClientConfig webClientConfig;
+    private final GameMapper gameMapper;
     @Value("${gameyfin.igdb.api.client-id}")
     private String clientId;
-
     @Value("${gameyfin.igdb.api.client-secret}")
     private String clientSecret;
-
     @Value("${gameyfin.igdb.config.preferred-platforms:6}")
-    private String preferredPlatforms;
-
+    private Integer[] preferredPlatforms;
```

Comment:
Personally I prefer List<> instead of Array, is there any reason here to not use List<>?

Reply:
Oh no not at all, it's just a residual change because the initial implementation of the IgdbApiQueryBuilder didn't support Lists. I'll change this.